pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
+| $$MaxP$$ | MW | Maximum generator active power output |
+| $$ReactiveLimits$$ | Mvar | Operational limits of the generator (P/Q/V diagram) |
+| $$RatedS$$ | MVA | The rated nominal power |
 | $$TargetP$$ | MW | The active power target |
 | $$TargetQ$$ | MVAr | The reactive power target |
 | $$TargetV$$ | kV | The voltage target |
-| $$RatedS$$ | MVA | The rated nominal power |
-| $$ReactiveLimits$$ | - | Operational limits of the generator (P/Q/U diagram) |
+| $$RegulatingTerminal$$ |  | Associated node or bus for which voltage is to be regulated |
+| $$VoltageRegulatorOn$$ |  | True if the generator regulates voltage |
+| $$EnergySource$$ |  | The energy source harnessed to turn the generator |
 
-**Specifications**
-- The minimal active power (in $$MW$$), expected to be lower than the maximal active power. The target $$P$$ is necessarily comprised between the two.
-- Setpoints for generators ($$targetV$$, $$targetP$$ and $$targetQ$$):
-    - They follow the generator sign convention: a positive value of $$targetP$$ means an injection into the bus.
-    - A positive value for the $$targetP$$ and the $$targetQ$$ means a negative value at the corresponding terminal (which is in passive-sign convention).
-- A set of reactive limits can be associated to a generator. All the reactive limits modelings available in the library are described [here](#reactive-limits).
-- the rated nominal power (MVA)
-<span style="color:red"> TODO: explain what it is.</span>
-
-- Either the generator is regulating the voltage, and the voltage setpoint is required, or it is not regulating and the reactive power setpoint is required instead.
-
-**Metadata**    
-A generator in IIDM comprises some metadata:
-- the energy source, which can be:
-    - `HYDRO`
-    - `NUCLEAR`
-    - `WIND`
-    - `THERMAL`
-    - `SOLAR`
-    - `OTHER`
-- The participation to regulation (through a boolean)
-- The regulating terminal, which can be local or remote: it is the specific connection point on the network where the setpoint is measured.
+The values `MinP`, `MaxP` and `TargetP` are required. The minimum active power output can not be greater than the maximum active power output. `TargetP` must be inside this active power limits. `RatedS` specifies the nameplate apparent power rating for the unit, it is optional and should be a positive value if it is defined. The reactive limits of the generator (`ReactiveLimits`) are optional, if they are not given the generator is considered with unlimited reactive power. Reactive limits can be given as a pair or min/max values or as a  reactive capability curve.
 
-**Available extensions**
+The `VoltageRegulatorOn` attribute is required. It voltage regulation is enabled, then `TargetV` and `RegulatingTerminal` must also be defined. If the voltage regulation is disabled, then `TargetQ` is required. `EnergySource` is optional, it can be: `HYDRO`, `NUCLEAR`, `WIND`, `THERMAL`, `SOLAR` or `OTHER`.
```

Comment:
From my point of view, it's too close to the java model.

Reply:
Yes, too technical. Some functional explanations have to be added to understand how voltage regulation works. We have developpers and users that know nothing about how a network works.